pkg/build/types/parser/deffile.go

```diff
@@ -131,51 +131,101 @@ func getSectionName(line string) string {
 	return lineSplit[0]
 }
 
-// parseTokenSection splits the token into maximum 2 strings separated by a newline,
-// and then inserts the section into the sections map
-//
-func parseTokenSection(tok string, sections map[string]string) error {
+// parseTokenSection into appropriate components to be placed into a types.Script struct
+func parseTokenSection(tok string, sections map[string]*types.Script, files *[]types.Files) error {
 	split := strings.SplitN(tok, "\n", 2)
 	if len(split) != 2 {
 		return fmt.Errorf("Section %v: Could not be split into section name and body", split[0])
```

Comment:
Return error should have no caps.

```suggestion
 		return fmt.Errorf("section %v: could not be split into section name and body", split[0])
```